.ci/diff-report.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+case $1 in
+
+# Downloads all files necessary to generate regression report.
+download-files)
+  checkForVariable "GITHUB_TOKEN"
+  echo "Downloading files..."
+
+  # check for projects link from PR, if not found use default from contribution repo
+  LINK="${LINK_FROM_PR:-$DEFAULT_PROJECTS_LINK}"
+
+  # get projects file
+  curl --fail-with-body -X GET "${LINK}" \
+    -H "Accept: application/vnd.github+json" \
+    -H "Authorization: token $GITHUB_TOKEN" \
+    -o project.properties
+
+  if [ -n "$NEW_MODULE_CONFIG_LINK" ]; then
+    curl --fail-with-body -X GET "${NEW_MODULE_CONFIG_LINK}" \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: token $GITHUB_TOKEN" \
+      -o new_module_config.xml
+  fi
+
+  if [ -n "$DIFF_CONFIG_LINK" ]; then
+    curl --fail-with-body -X GET "${DIFF_CONFIG_LINK}" \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: token $GITHUB_TOKEN" \
+      -o diff_config.xml
+  fi
+
+  if [ -n "$PATCH_CONFIG_LINK" ]; then
+    curl --fail-with-body -X GET "${PATCH_CONFIG_LINK}" \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: token $GITHUB_TOKEN" \
```

Comment:
You need to pass it from GitHub action config

https://github.com/checkstyle/checkstyle/blob/82cdd1ed0dac91e7cfa749fca161df69d51949e2/.github/workflows/release-new-milestone-and-issues-in-other-repos.yml#L51

Reply:
Followed format from elsewhere in this file with uppercase GITHUB_TOKEN